ci/jenkins/test-vmc.sh

```diff
@@ -629,6 +817,51 @@ function cleanup_cluster() {
     release_static_ip
 }
 
+function cleanup_aws_cluster() {
+    echo "=== Cleaning up AWS cluster ${CLUSTER} ==="
+    export KUBECONFIG=$KUBECONFIG_PATH
+
+    echo "=== Cleaning up Resources ===="
+    export AWS_CONTROLLER_IAM_ROLE=$AWS_SERVICE_USER_ROLE_ARN
+    clusterctl delete --infrastructure aws
+    export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile --region $AWS_REGION)
+    clusterctl init --infrastructure aws
+    
+    kubectl delete cluster ${CLUSTER} -n ${CLUSTER}
+
+    echo "=== Cleaning up Instances ==="
+    instance_ids=$(aws ec2 describe-instances --filters "Name=tag:Name,Values=${CLUSTER}-*" --query "Reservations[*].Instances[*].InstanceId" --output text)
+    for instance_id in $instance_ids; do
+        aws ec2 terminate-instances --instance-ids "$instance_id"
+    done
+
+    echo "=== Cleaning up Load Balancer ==="
+    loadbalancer_name=$(aws elb describe-load-balancers --query "LoadBalancerDescriptions[?SourceSecurityGroup.GroupName=='${CLUSTER}-apiserver-lb'].LoadBalancerName" --output text)
+    aws elb delete-load-balancer --load-balancer-name ${loadbalancer_name}
+    sleep 90s
+    
+    echo "=== Cleaning up Security Groups ==="
```

Comment:
Can it keep the security groups? In case some clusters are created at the same time, the security group may be used by other clusters.